src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -3359,51 +3512,6 @@ pub const IFDR_MSG_SIZE: ::c_int = 64;
 pub const IFDR_REASON_MSG: ::c_int = 1;
 pub const IFDR_REASON_VENDOR: ::c_int = 2;
 
-// sys/net/if_mib.h
-
-/// non-interface-specific
-pub const IFMIB_SYSTEM: ::c_int = 1;
```

Comment:
This is a breaking change btw (moving a constant to a different module).

Reply:
Yes, this is handled separately as #3367, on which this PR is based